src/test/java/it/gdhi/service/CountryHealthDataServiceTest.java

```diff
@@ -312,7 +312,7 @@ public void shouldGetAdminViewFormDetailsForCurrentYear() {
     }
 
     @Test
-    public void shouldReturnTrueIfAllFieldsAreValid() {
```

Comment:
shouldReturnTrueIfAllFieldsAreValid
shouldReturnFalseIfGovtApprovedIsTrueAndJustApproverEmailIsMissing


can use when instead of If :
shouldReturnTrueIfAllFieldsAreValid ->> shouldReturnTruewhenAllFieldsAreValid